common/services/person.js

```diff
@@ -2,6 +2,40 @@ const { mapKeys, mapValues, uniqBy, omitBy, isNil } = require('lodash')
 
 const apiClient = require('../lib/api-client')()
 
+const unformat = require('./person-unformat')
+
+const relationshipKeys = ['gender', 'ethnicity']
+const identifierKeys = [
+  'police_national_computer',
+  'criminal_records_office',
+  'prison_number',
+  'niche_reference',
+  'athena_reference',
+]
+const dateKeys = ['date_of_birth']
+const assessmentKeys = [
+  // court
+  'solicitor',
+  'interpreter',
+  'other_court',
+  // risk
+  'violent',
+  'escape',
+  'hold_separately',
+  'self_harm',
+  'concealed_items',
+  'other_risks',
+  // health
+  'special_diet_or_allergy',
+  'health_issue',
+  'medication',
+  'wheelchair',
+  'pregnant',
+  'other_health',
+  'special_vehicle',
+]
+const explicitAssessmentKeys = ['special_vehicle']
```

Comment:
I think there is another that is explicit - `not_to_be_released`

Reply:
Added - I think it might be better to add these as properties of the field themselves. It will also make it easier to test. But I'll leave that for a refactor another day...